### PR TITLE
Add beanstalkd scaler docs

### PR DESCRIPTION
**Pull request by sbarnesthornton**
Adds docs regarding the beanstalkd scaler introduced in [#5901](https://github.com/kedacore/keda/pull/6081).

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/5901
Relates to https://github.com/kedacore/keda/pull/6081

### CODE REVIEW

File: content/docs/2.15/scalers/beanstalkd.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
++++
+title = "Beanstalkd"
+availability = "v2.15+"
```
Comment (tomkerkhove) [inline content/docs/2.15/scalers/beanstalkd.md:3]: Can you please update this to 2.16 and move to the new folder please?

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.15/scalers/beanstalkd.md (hunk 1 above)
Comment (sbarnesthornton) [reply content/docs/2.15/scalers/beanstalkd.md]: Thanks for the review. All updated to 2.16.

**Comment by sbarnesthornton** [comment]
The test deployment preview is failing after I have changed the version to be 2.16, which seems potentially something to do with caching as it is still looking for the docs in 2.15. I'm not sure how to sort this, but it was working before the move to 2.16.

File: content/docs/2.16/scalers/beanstalkd.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+```
+
+**Parameter list:**
+
+- `server` - Address of beanstalkd server `<host>:<port>`. If no port is specified then the scaler will default to `11300`.
+- `includeDelayed` - Whether to include delayed jobs in the count used for scaling. Defaults to false so only `ready` and `reserved` jobs are counted.
+- `tube` - Name of the tube to scale on.
+- `activationValue` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds). (Default: `0`, Optional, This value can be a float)
+- `value` - Number of jobs in the queue to trigger on. Defaults to `ready`+`reserved` jobs if `includeDelayed` isn't set.
+- `timeout` - Timeout in seconds for establishing a connection to the beanstalkd server. Defaults to 30.
```
Comment (JorTurFer) [inline content/docs/2.16/scalers/beanstalkd.md:33]:
```suggestion
- `timeout` - Timeout in seconds for establishing a connection to the beanstalkd server. (Default: `30`, Optional)
```

**Review by JorTurFer — APPROVED** [review]
LGTM! ❤️ 

Could you add the page to this file? https://github.com/kedacore/keda-docs/blob/main/.htmltest.yml 

As the scaler doesn't appear in v2.15, the links checker is detecting it as failure

File: content/docs/2.16/scalers/beanstalkd.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+triggers:
+  - type: beanstalkd
+    metadata:
+      server: beanstalkd.internal-namespace:11300
+      includeDelayed: "false"
+      tube: "default"
+      activationValue: "10"
+      value: "15"
+      timeout: "30"
+
```
Comment (zroubalik) [inline content/docs/2.16/scalers/beanstalkd.md:23]:
```suggestion
```
nit

**Review by zroubalik — APPROVED** [review]
LGTM

**Pull request merged**